src/opencmiss/utils/maths/vectorops.py

```diff
@@ -67,49 +69,27 @@ def rotmx(quaternion):
     return mx
 
 
-def mxconstantmult(m, c):
+def matrixconstantmult(m, c):
```

Comment:
I think these sort of function names should be changed to `matrix_constant_mult` for consistency across the API.  We can always assign alternative names at the bottom of the file and print out a message saying deprecated.

Reply:
Fixed.